.stylelintrc

```diff
@@ -1,6 +1,9 @@
 {
   "extends": "stylelint-config-recommended",
   "processors": ["@mapbox/stylelint-processor-arbitrary-tags"],
+  "plugins": [
```

Comment:
Do we need to do similar changes in `core/templates/css/.stylelintrc`? [I'll approve this PR once  this comment is resolved!]

Reply:
I have already made this change there.

Reply:
Ah, I see!